### PR TITLE
feat: add streaming ui for camera and sync the style with iOS.

### PR DESCRIPTION
**Pull request by jasonchuang**
1. Add stats update ui in the right, bottom corner
2. UI style ref: [ios_stream_stats](https://redmine.livehouse.in/attachments/download/6617/demo_app_streaming_stats.png)
3. formatBitrate refs: [Formatter](https://android.googlesource.com/platform/frameworks/base/+/ics-mr1/core/java/android/text/format/Formatter.java#47)

### CODE REVIEW

File: Streaming/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -21,4 +21,11 @@
     <string name="video_quality_360p">360p</string>
     <string name="video_quality_480p">480p</string>
     <string name="video_quality_720p">720p</string>
+
+    <!-- Streaming Statistics Report -->
+    <string name="bps">bps</string>
+    <string name="kiloBps">Kbps</string>
```
[the remark this answers is not in the document]
Comment (jasonchuang) [reply Streaming/src/main/res/values/strings.xml]: 這邊是照抄 [frameworks](https://android.googlesource.com/platform/frameworks/base/+/ics-mr1/core/res/res/values/strings.xml#19)
在討論看看要不要改成有底線的格式會比較一致

File: Streaming/src/main/java/io/straas/android/sdk/streaming/demo/Utils.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import android.content.Context;
+
+import io.straas.android.sdk.demo.R;
+import io.straas.android.sdk.streaming.StreamStatsReport;
+
+public class Utils {
+
+    public static String toDisplayText(Context context, StreamStatsReport streamStatsReport) {
+        String bitrate = formatBitrate(context, streamStatsReport.getBitrate());
+        return  String.format("%s\n%.1f fps", bitrate,
```
[the remark this answers is not in the document]
Comment (jasonchuang) [reply Streaming/src/main/java/io/straas/android/sdk/streaming/demo/Utils.java]: ok

File: Streaming/src/main/java/io/straas/android/sdk/streaming/demo/Utils.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+import android.content.Context;
+
+import io.straas.android.sdk.demo.R;
+import io.straas.android.sdk.streaming.StreamStatsReport;
+
+public class Utils {
+
+    public static String toDisplayText(Context context, StreamStatsReport streamStatsReport) {
+        String bitrate = formatBitrate(context, streamStatsReport.getBitrate());
```
[the remark this answers is not in the document]
Comment (jasonchuang) [reply Streaming/src/main/java/io/straas/android/sdk/streaming/demo/Utils.java]: ok

File: Streaming/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -21,4 +21,11 @@
...
+    <string name="megaBps">Mbps</string>
+    <string name="gigaBps">Gbps</string>
+    <string name="bitrateSuffix">"<xliff:g id="number" example="123">%1$s</xliff:g> <xliff:g id="unit" example="Kbps">%2$s</xliff:g>"</string>
```
[the remark this answers is not in the document]
Comment (jasonchuang) [reply Streaming/src/main/res/values/strings.xml]: 要, 這邊同上, 而且 xliff 格式比較好讀一點

File: Streaming/src/main/java/io/straas/android/sdk/streaming/demo/Utils.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (ItsRay) [reply Streaming/src/main/java/io/straas/android/sdk/streaming/demo/Utils.java]: 變數，看有沒有需要

**Pull request merged**